bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/binding/BaseThingHandler.java

```diff
@@ -189,7 +189,7 @@ public void channelUnlinked(ChannelUID channelUID) {
     protected void validateConfigurationParameters(Map<String, Object> configurationParameters)
             throws ConfigValidationException {
         ThingType thingType = TypeResolver.resolve(getThing().getThingTypeUID());
-        if (thingType != null) {
+        if (thingType != null && thingType.getConfigDescriptionURI() != null) {
```

Comment:
The result (and the test that is done) is the same, but as long the ThingType provides the function "hasConfigDescriptionURI" I would prefer to use that function. :wink: